proposals/0143-conditional-conformances.md

```diff
@@ -291,6 +291,20 @@ general) than the constraints `T: S`, because every `S` is also an
 extension Y: P where T: R { }
 ```
 
+## Standard library adoption
+
+Adopt conditional conformances to make various standard library types
+that already have a suitable `==` conform to `Equatable`. Specifically:
+
+```swift
+extension Optional: Equatable where Wrapped: Equatable { /*== already exists */ }
+extension Array: Equatable where Element: Equatable { /*== already exists */ }
+extension ArraySlice: Equatable where Element: Equatable { /*== already exists */ }
+extension ContiguousArray: Equatable where Element: Equatable { /*== already exists */ }
+extension Dictionary: Equatable where Value: Equatable { /*== already exists */ }
+extension Set: Equatable { /*== already exists */ }
```

Comment:
This is missing `where Value: Equatable`, right?

Reply:
Set's members already need to be Hashable, so `Set` is always `Hashable` and `Equatable`. This one should be unnecessary.

Reply:
Right, `Set` could have been `Equatable` at any point. I think we didn't mark it as such because it would have felt out of place without `Array` or `Dictionary` conforming to `Equatable`.

I'll push an update shortly to clarify this, because it's really confusing as written.

Reply:
Oh of course.

Reply:
@jckarter points out that `Set` has always been `Hashable`, so there's actually no change here! I've clarified correctly this time.